packages/babel-types/src/definitions/es2015.js

```diff
@@ -69,8 +57,9 @@ defineType("ArrowFunctionExpression", {
     ...functionCommon,
     ...functionTypeAnnotationCommon,
     expression: {
```

Comment:
This is actually stale code. We removed this field in #6773, we should drop this too.